### PR TITLE
MP-6055 Marketplace Shopping Lists feature walkthrough added.

### PR DESCRIPTION
**Pull request by volodymyr-bezuhlyi**
## PR Description

Ticket: https://spryker.atlassian.net/browse/MP-6055

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Marketplace Shopping Lists feature walkthrough
+description: The Marketplace Shopping Lists feature lets you create and manage Marketplace Shopping Lists.
+template: feature-walkthrough-template
+---
+
+By using the 'Marketplace Shopping Lists' feature, shoppers can add a Merchant Product or a Merchant Offer from PDP to Shopping Lists. Then shoppers can add a Merchant Product or a Merchant Offer from the Shopping List to the Cart so they can be bought.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:7]:
```suggestion
The 'Marketplace Shopping Lists' feature allows customers to create and share multiple lists of merchant products and product offers between company business units or single users. Shopping lists can be shared between users with different sets of permissions.
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Module dependency graph
+
+The following diagram illustrates the dependencies between the modules for the {Feature name} feature.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:11]:
```suggestion
The following diagram illustrates the dependencies between the modules for the 'Marketplace Shopping Lists' feature. 
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/b9b242da-b56f-452d-b44f-7eb740adf1da.png?utm_medium=live&utm_source=custom)
+
+| MODULE                                   | DESCRIPTION                                                                 |
+|------------------------------------------|-----------------------------------------------------------------------------|
+| ProductOfferShoppingListWidget           | Provides widgets for product offer integration into a shopping list.        |
+| ProductOfferShoppingListDataImport       | Provides Data Import for Shopping Lists with Product Offers.                |
+| ProductOfferShoppingList                 | Provides product offer functionality for Shopping List.                     |
+| MerchantProductShoppingListsRestApi      | Provides REST API endpoints to manage merchant product shopping lists.      |
+| MerchantProductOfferShoppingListsRestApi | Provides REST API endpoints to manage merchant product offer shopping list. |
+| ProductOfferShoppingListsRestApi         | Provides REST API endpoints to manage product offer shopping list.          |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:22]: - ShoppongList?

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Domain model
+
+The following schema illustrates the Marketplace Shipment domain model:
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:26]:
```suggestion
The following schema illustrates the 'Marketplace Shopping Lists' domain model:
```

File: docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: "File details: product_offer_shopping_list.csv"
+last_updated: May 13, 2022
+description: This document describes the product_offer_shopping_list.csv file to configure merchant product offer shopping list in your Spryker shop.
+template: import-file-template
+---
+
+This document describes the `product_offer_shopping_list.csv` file to configure [Marketplace Shopping Lists](/docs/marketplace/user/features/{{page.version}}/marketplace-shopping-lists-feature-walkthrough.html#marketplace-shopping-lists) information in your Spryker shop.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md:8]:
```suggestion
This document describes the `product_offer_shopping_list.csv` file to configure shopping lists with product offers in your Spryker shop.
```

File: docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+title: "File details: product_offer_shopping_list.csv"
+last_updated: May 13, 2022
+description: This document describes the product_offer_shopping_list.csv file to configure merchant product offer shopping list in your Spryker shop.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md:4]:
```suggestion
description: This document describes the product_offer_shopping_list.csv file to configure shopping lists with product offers in your Spryker shop.
```

File: docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md (hunk 5 above)
Comment (ilyakubanov) [reply docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md:8]: `shopping lists with product offers` - correct link, for now it doesn't work

File: docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## Import file parameters
+
+The file should have the following parameters:
+
+| PARAMETER               | REQUIRED? | TYPE    | DEFAULT VALUE | REQUIREMENTS OR COMMENTS | DESCRIPTION                                                                                                                                                 |
+|-------------------------|-----------|---------|---------------|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| shopping_list_item_key  | &check;   | String  |               | Unique                   | The key that will identify the shopping list item to add data to.                                                                                           |
+| product_offer_reference | &check;   | String  |               | Unique                   | Identifier of the [merchant product offer](/docs/marketplace/user/features/{{site.version}}/marketplace-product-offer-feature-overview.html) in the system. |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md:23]: why unique? can't we add different offers to the same shopping list or the same offer for different shopping lists?

Comment (ilyakubanov) [inline docs/marketplace/dev/data-import/202204.0/file-details-product-offer-shopping-list.csv.md:23]:
```suggestion
| shopping_list_item_key  | &check;   | String  |               | Unique                   | Identifier of the merchant in the system.                                                                                           |
| product_offer_reference | &check;   | String  |               | Unique                   | Identifier of the [merchant product offer](/docs/marketplace/user/features/{{site.version}}/marketplace-product-offer-feature-overview.html) in the system. |
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+title: Marketplace Shopping Lists feature walkthrough
+description: The Marketplace Shopping Lists feature allows customers to create and share multiple lists of merchant products and product offers between company business units or single users.
+template: feature-walkthrough-template
+---
+
+The 'Marketplace Shopping Lists' feature allows customers to create and share multiple lists of merchant products and product offers between company business units or single users. Shopping lists can be shared between users with different sets of permissions.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:7]:
```suggestion
The `Marketplace Shopping Lists` feature allows customers to create and share multiple lists of merchant products and product offers between company business units or single users. Shopping lists can be shared between users with different sets of permissions.
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+{% info_block warningBox "User documentation" %}
+
+To learn more about the feature and to find out how end users use it, see [Marketplace Shopping List feature overvie](/docs/marketplace/user/features/{{page.version}}/marketplace-shopping-list-feature-overview.html) for business users.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:11]:
```suggestion
To learn more about the feature and to find out how end users use it, see [Marketplace Shopping List feature overview](/docs/marketplace/user/features/{{page.version}}/marketplace-shopping-list-feature-overview.html) for business users.
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+| ProductOfferShoppingList                 | Provides product offer functionality for Shopping List.                                                                                   |
+| MerchantProductShoppingListsRestApi      | Provides REST API endpoints to manage merchant product shopping lists.                                                                    |
+| MerchantProductOfferShoppingListsRestApi | Provides REST API endpoints to manage merchant product offer shopping list.                                                               |
+| ProductOfferShoppingListsRestApi         | Provides REST API endpoints to manage product offer shopping list.                                                                        |
+| ShoppingList                             | Provides infrastructure and functionality to handle multiple shopping lists for a customer account as well as manage shopping list items. |
+
+## Domain model
+
+The following schema illustrates the 'Marketplace Shopping Lists' domain model:
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:33]:
```suggestion
The following schema illustrates the `Marketplace Shopping Lists` domain model:
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+{% info_block warningBox "User documentation" %}
+
+To learn more about the feature and to find out how end users use it, see [Marketplace Shopping List feature overvie](/docs/marketplace/user/features/{{page.version}}/marketplace-shopping-list-feature-overview.html) for business users.
+
+{% endinfo_block %}
+
+## Module dependency graph
+
+The following diagram illustrates the dependencies between the modules for the 'Marketplace Shopping Lists' feature.
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:17]:
```suggestion
The following diagram illustrates the dependencies between the modules for the `Marketplace Shopping Lists` feature.
```

File: docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+|                                                                                                                                                                         |                                                                                                                                                      |                                                                                                                                                                    | [Managing shopping list items](/docs/marketplace/dev/glue-api-guides/{{page.version}}/marketplace-shopping-lists/managing-marketplace-shopping-list-items.html) |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md:43]: - glue API IG link is missing
- please carefully recheck all the linked docs exist in related PRs

**Review by ilyakubanov — APPROVED** [review]
Broken links should work after https://github.com/spryker/spryker-docs/pull/798 and https://github.com/spryker/spryker-docs/pull/827 release

**Review by AdvaBD — APPROVED** [review]

**Review by AdvaBD — DISMISSED** [review]

**Review by AdvaBD — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**